fencing/commands.c

```diff
@@ -252,6 +252,24 @@ create_async_command(xmlNode * msg)
     return cmd;
 }
 
+static int
+get_action_limit(stonith_device_t * device)
+{
+    const char *value = NULL;
+    int action_limit = 1;
+
+    value = g_hash_table_lookup(device->params, STONITH_ATTR_ACTION_LIMIT);
+    if (value) {
+       action_limit = crm_parse_int(value, "1");
+       if (action_limit == 0) {
```

Comment:
Given the second parameter of  crm_parse_int(), it should return the specified default "1" in case of an invalid string. Actually any configured negative values would be unlimited.  I think I was trying to make this consistent with migration-limit, batch-limit, node-action-limit and pe-*-series-max, which don't use INFINITY for unlimited either.